### DISCUSSION TITLE
Posso definir outra origem de uma página hospedada no GitHub?

### DISCUSSION THREAD

**Discussion by Gusta361** [community/community#133055] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Nesse caso, definir uma pasta como **"pages"** ou algum outro nome personalizado no lugar de **"master"** ou **"docs"** por exemplo.

**Comment by itvinl** [comment, marked as answer]
**Olá amigo!! Há um artigo do GitHub que explica essa questão, caso queira se aprofundar melhor nessa questão:**

https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/ 

**_Porém, para simplificar, segundo o artigo até então, não é possível definir uma pasta ou branch personalizada._**
